amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -184,18 +185,23 @@ class TableDetail extends React.Component<TableDetailProps & RouteComponentProps
                       <WatermarkLabel watermarks={ data.watermarks }/>
                     </>
                   }
+                  {
+                    !!data.last_updated_timestamp &&
+                    <>
+                      <div className="section-title title-3">Last Updated</div>
+                      <div className="body-2">{ formatEpochTime(data.last_updated_timestamp) }</div>
```

Comment:
Our mocks seem to specify a new format: 
![image](https://user-images.githubusercontent.com/1790900/74365750-896e8e80-4d83-11ea-8092-13f2048d7308.png)

I'm fine with leaving it as is to complete the task, but I think we should check whether or not this new format was intentionally chosen by Design and understand if this new format is a must-have, nice-to-have, or if it was just placeholder text.

Reply:
This is an easy fix, but this would bring us to 3 unique date formats. I'd prefer to combine this with the one used in the footer. Does anyone really need to know the exact minute and second that the amundsen update job finished (or is that started?)

Reply:
My personal thought is that seconds may not matter, but I don't feel a need to force us into have the least amount of date formats possible. We could always end up with more for different use cases. In addition the code base could end up including a whole mapping of multiple date formats anyway if the community wants to introduce better internationalization support.

My vote would be to leave the footer as is, but run it by design or users if you feel strongly.

Reply:
That's a good point about internationalization. In that case we should create a configs for a date formats and use that instead of putting it in a constants file. Ideally, we'd have just two date formats, maybe three. I'd just hate to see different date formats used randomly in various locations in the same product.